packages/headless/src/api/platform-client.test.ts

```diff
@@ -28,11 +29,47 @@ describe('url helper', () => {
 
   it('return the correct #analyticsUrl()', () => {
     allValidPlatformCombination().forEach((expectation) => {
-      expect(analyticsUrl(expectation)).toEqual(
-        expectation.analytics.split('/rest')[0]
-      );
+      if (!expectation.multiRegionSubDomain) {
```

Comment:
nitpik: conditional expectation is a code smell
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md
I would  rework this with a `it.each` instead to move the loop outside the it